CodeBeam.MudBlazor.Extensions/Components/SignaturePad/MudSignaturePad.razor

```diff
@@ -0,0 +1,58 @@
+﻿@using Microsoft.JSInterop
+@using MudBlazor.Charts
+@using MudBlazor.Utilities
+@using System.Text
+@namespace MudExtensions
+@inject IJSRuntime JsRuntime
+@inherits ComponentBase
+
+<MudPaper Elevation="@Elevation" Class="@OuterClass">
+    <div style="@CanvasContainerStyle" class="@CanvasContainerClass">
+        <canvas @ref="_reference" id="@_id"></canvas>
+    </div>
+    <MudToolBar Style="@ToolbarStyle" Class="my-2">
+        <MudStack Row="true" Class="justify-content-between align-center">
+            @if (ShowLineWidth)
```

Comment:
We can add a `ToolbarContent` as renderfragment parameter. Maybe users want to add their buttons, pickers etc.